src/flow_solver/flow_solver_cases/flow_solver_case_base.cpp

```diff
@@ -151,7 +152,11 @@ template class FlowSolverCaseBase<PHILIP_DIM,PHILIP_DIM>;
 #endif
 
 #if PHILIP_DIM!=1
-template class FlowSolverCaseBase<PHILIP_DIM,PHILIP_DIM+2>;
```

Comment:
You could leave this and add `#if PHILIP_DIM==2 template class FlowSolverCaseBase<PHILIP_DIM,1>; #endif`

Reply:
Yeah, but 3 #if preprocessor statements (dim==1, dim!=1, dim==2) might make it hard for readability. And it looks like the flow solver will be used for several test cases in the future. Maybe it's better to remove all #if statements (because we don't need to template with MeshTypes) and template this class as  `FlowSolverCaseBase<PHILIP_DIM, nstate>`, with nstate from 1 to 5 to cover all possible cases.